noplan-inc/nft.storage-rs
# Implement `nft_storage_cli` and Update Examples in `nft_storage_core`

## Changes Proposed:
- Developed a new command-line interface (CLI) tool named `nft_storage_cli`, utilizing `nft_storage_core`.
- Added new examples in the `nft_storage_core` module to demonstrate its functionality.
- Updated READMEs for both `nft_storage_core` and `nft_storage_cli` to reflect the latest changes and usage instructions.

## Implementation Details:
- The `nft_storage_cli` tool is designed to provide a user-friendly interface for interacting with the functionalities offered by `nft_storage_core`.
- The added examples in the `nft_storage_core` module cover a range of use cases, providing clear and practical demonstrations of how to use the core functionalities.
- READMEs have been revised to include updated information, installation guidelines, and example usages for both modules.

## How to Test:
1. Clone the repository and switch to the relevant branch.
2. Follow the installation instructions in the README of `nft_storage_cli`.
3. Test the CLI tool using the provided command examples.
4. Review the new examples in `nft_storage_core` for clarity and comprehensiveness.
5. Ensure all documentation is clear and provides sufficient guidance for new users.
